### ISSUE TITLE
xnatph(stoichiometry) ratio maximum of 999999

### ISSUE THREAD

**Issue by CharlesCardot** [times-software/feff10#18]
Given how FEFF writes the pot.inp file, stoichiometries of 10^6 or larger end up spilling over into the lmaxsc column.

**feff.inp**
![image](https://user-images.githubusercontent.com/43706411/206276202-b54f4ae0-06ac-4282-9daf-cfc30453655d.png)

**pot.inp**
![image](https://user-images.githubusercontent.com/43706411/206276386-239c6c41-e3b0-4dcc-84f3-f47ed5fef2fb.png)

A quick fix for this is to scale down all the stoichiometry values by dividing them all by 10. 
4185 -> 418.5
14570 -> 1457.0
135 -> 13.5
4810 -> 481.0
And leave the Nitrogen stoichiometry alone. This does change the overall ratio, but it's up to the user if a ratio of 10^5 vs a ratio of 10^6 really makes a difference.

For a more permanent, but still a Band-Aid fix, I also tried changing line 592 in /src/COMMON/m_inpmodules.f90 such that the string formatting had more digits to work with when pot.inp gets written.
![image](https://user-images.githubusercontent.com/43706411/206278646-d3683052-2a8f-4c00-9e02-cb450e8a4802.png)
to
![image](https://user-images.githubusercontent.com/43706411/206278748-86874de9-4342-4ecc-9c99-f0941e21271b.png)

**Comment by jjkas** [comment]
This is correct, with the following caveat. FEFF has a minimum stoichiometry, which is 0.01. Thus the exact statement would be:
 
If the minimum stoichiometry in the feff.inp file is >= 0.01, then
the ratio of the maximum stoichiometry to the minimum stoichiometry must be less than 10^7 (999999 or less).

If the minimum stoichiometry in the feff.inp file is < 0.01, then
the ratio of the maximum stoichiometry to 0.01 must be less than 10^7 (999999 or less).
